.gitignore

```diff
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vauxoo/OCA Mantainer Tools
+.eslintrc.json
+.flake8
+.pre-commit-config*
+.pylintrc*
```

Comment:
Done. File has a newline but for some reason Github does not show it. If you clone the repo locally or edit the file on Github it will show a newline at the end of the file.